src/parser_urdf.cc

```diff
@@ -1604,6 +1604,7 @@ void CopyBlob(tinyxml2::XMLElement *_src, tinyxml2::XMLElement *_blob_parent)
 void InsertSDFExtensionCollision(tinyxml2::XMLElement *_elem,
                                  const std::string &_linkName)
 {
+  bool link_found = false;
```

Comment:
nit: our style guide uses `camelCase` for variables. Can you update this and other variables in this PR?